openfifth/aspen-discovery
# See all locations enhancement

Patch enhancing the 'Where is it?' modal by adding edition information as well as book cover thumbnails, as request in [ticket](https://app.plane.so/ptfs-europe/projects/b24083da-a488-4563-a393-9101dacefd66/issues/)

**Test plan**

- in Aspen Administration > Catalog/Grouped Works > Grouped Work Display > <setting in use>,  toggle on the 'Show Covers for Editions' checkbox
- it may be work also ensuring that the "Show Where is it?" option is set to either 'Show first 3 available copies & Where Is It link always' or  'Show Where Is It link only'
- run a search
- open the 'Where is it?' link
- notice that the modal that opens includes thumbnails and edition information
- in Aspen Administration > Catalog/Grouped Works > Grouped Work Display > <setting in use>,  toggle off the 'Show Covers for Editions' checkbox
- run a search
- open the 'Where is it?' link
- notice that the modal that opens includes edition information, but no thumbnails

If testing without the patch, then neither the edition information nor the thumbnails will show
